README.md

```diff
@@ -1 +1,249 @@
-# cozy-homes-back-end
\ No newline at end of file
+
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍